preprocessing/science/rconv/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ if (NOT PROJ4_FOUND)
         pkg_check_modules(PROJ4 QUIET proj>=4.9)
         if (PROJ4_FOUND)
             set(CMAKE_INSTALL_RPATH "${PROJ4_LIBRARY_DIRS};${CMAKE_INSTALL_RPATH}")
+            set(PROJ4_LIBRARIES "${PROJ4_LINK_LIBRARIES}")
```

Comment:
This is a bit random - was this tested or just committed by accident?

Reply:
Hi, yes. It was tested. There is a difference how pkg-config and cmake name LINK_LIBRARIES. I had to fix it for the Docker CI image.